webpay/pay/views.py

```diff
@@ -147,6 +150,7 @@ def configure_transaction(request, data=None):
                     .format(mcc=mcc, mnc=mnc))
 
     is_simulation = request.session.get('is_simulation', False)
+    payment_required = request.session.get('payment_required', True)
```

Comment:
if you check the ``pricePoint`` here then you wouldn't have to set anything in the session